### ISSUE TITLE
2.3.0-beta.2-linux-x86_64.AppImage - NO PROJECTS IN WORKSPACE displayed

### ISSUE THREAD

**Issue by SvenMeyer** [trufflesuite/ganache#1621]
2.3.0-beta.2-linux-x86_64.AppImage
Arch Linux
[ganache-20200208-182528.log.gz](https://github.com/trufflesuite/ganache/files/4173894/ganache-20200208-182528.log.gz)

- AppImage start
- Quickstart
- Settings > changed mnemonic
- [Save and Restart]

- [SAVE]
- Workspace name changes from 'Quickstart' to 'TAME-LIQUID'

- 18:23 Settings > Advanced > VERBOSE LOGS
- [Save and Restart]

- CONTRACS Tab
- shows 'No Projects in Workspace'
- click [Link Truffle Projects]
- select truffle-config.js
- [Save and Restart]

- CONTRACS Tab
- shows 'No Projects in Workspace'

checking `/.config/Ganache/workspaces/tame-liquid/Settings` shows entry for the chosen project:
```json
{
  "server": {
    "gasLimit": 6721975,
    "gasPrice": 20000000000,
    "hardfork": "muirGlacier",
    "db_path": "/home/sum/.config/Ganache/workspaces/tame-liquid/chaindata",
    "default_balance_ether": 100,
    "hostname": "127.0.0.1",
    "locked": false,
    "mnemonic": "tomorrow draft giggle design purchase daring goddess cute inquiry giant thumb journey",
    "network_id": 5777,
    "port": 7545,
    "total_accounts": 10,
    "unlocked_accounts": [],
    "verbose": false,
    "vmErrorsOnRPCResponse": true
  },
  "flavor": "ethereum",
  "isDefault": false,
  "logDirectory": "/home/sum/Downloads/Ethereum/ganache",
  "name": "tame-liquid",
  "projects": ["/home/sum/DEV/IronOre/Proof-of-Asset-Eth/truffle-config.js"],
  "randomizeMnemonicOnStart": false,
  "uuid": "2122c5d2-1bb3-4fcf-9388-b819480f19f2",
  "verboseLogging": false
}
```

**Comment by davidmurdoch** [comment]
Thanks for opening all these issues, @SvenMeyer. I've confirmed this issue and triaged to be fixed in 2.3.0.

**Comment by davidmurdoch** [comment]
Fixed by 73a5df8abf3d8378cbaac2ba69768b57147d14a9
